esp/esp/tagdict/tests.py

```diff
@@ -5,10 +5,10 @@
 from esp.program.tests import ProgramFrameworkTest
 
 # Make test-only tags not raise warnings
-all_global_tags['test'] = (False, "")
-all_program_tags['test'] = (False, "")
-all_global_tags['test_bool'] = (True, "")
-all_program_tags['test_bool'] = (True, "")
+all_global_tags['test'] = (False, "", None, "manage", True)
+all_program_tags['test'] = (False, "", None, "manage", True)
+all_global_tags['test_bool'] = (True, "", False, "manage", True)
+all_program_tags['test_bool'] = (True, "", False, "manage", True)
```

Comment:
Why are the test-only tags "True" for showing on tag settings page?

Reply:
My thinking was that this would make them go through more of the new code for testing purposes (but of course they'll never actually show up on the real page).